lib/modules/dosomething/dosomething_user/dosomething_user.module

```diff
@@ -1337,13 +1337,28 @@ function dosomething_user_get_field_northstar_id($field_value) {
   return $field_value;
 }
 
+/**
+ * Return the specified Drupal user/global user's Northstar ID.
+ *
+ * @param  string $drupal_id
+ * @return mixed
+ */
+function dosomething_user_get_northstar_id($drupal_id = NULL) {
```

Comment:
Hmm post merge comment
But this function should be a handler or 'dosomething_user_get_field'
Not a stand alone function